### PR TITLE
Spinner

### PR DESCRIPTION
**Pull request by vsoch**
The loader itself is ugly as :helicopter: , but take a look at the functionality and let me know if it seems reasonable!

### CODE REVIEW

File: neurovault/apps/main/static/css/style.css [hunk 1]
```diff
@@ -211,3 +211,17 @@ ul.collection_nav li {
 .radio {
     padding-left:20px;
 }
+
+#fade {
+    display: none;
+    position:absolute;
+    top: 0%;
```
Comment (chrisfilo) [inline neurovault/apps/main/static/css/style.css]: Codacy Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/krzysztof-gorgolewski/NeuroVault_NeuroVault/file/837544010/issues/source?bid=2767031&fileBranchId=2795765#l218)

File: neurovault/apps/main/static/css/style.css [hunk 2]
```diff
@@ -211,3 +211,17 @@ ul.collection_nav li {
...
+    left: 0%;
```
Comment (chrisfilo) [inline neurovault/apps/main/static/css/style.css]: Codacy Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/krzysztof-gorgolewski/NeuroVault_NeuroVault/file/837544010/issues/source?bid=2767031&fileBranchId=2795765#l219)

Comment (vsoch) [inline neurovault/apps/main/static/css/style.css]: I'll probably be changing the color / spinner design entirely, I'm more interested in the functionality for this first look.

File: neurovault/apps/main/static/css/style.css (hunk 1 above)
Comment (vsoch) [inline neurovault/apps/main/static/css/style.css]: ahh codacy.com... and here I thought you turned into a robot.

**Comment by vsoch** [comment]
+1!

**Pull request merged**